### PR TITLE
add shawn suggestions

### PR DESCRIPTION
**Pull request by alvarj84**
This is a PR in order to keep track of the changes in the values file used by customer DSNY.

### CODE REVIEW

File: sysdig-values.yaml [hunk 1]
```diff
@@ -26,8 +26,8 @@ agent:
   #       proxy_user: smartproxydev
   #       proxy_password: dsnysv_137
   #       ssl: false
-  image:
-    tag: 12.3.1
+  # image:
+  #   tag: 12.3.1
```
Comment (shawnjohnson) [inline sysdig-values.yaml]:
```suggestion
```
We can remove the yaml for image/tag completely. No need to leave the commented out lines. Ideally in 99% of use-cases the customer should allow these values to be driven by the chart default values.

File: sysdig-values.yaml [hunk 2]
```diff
@@ -50,28 +50,28 @@ nodeAnalyzer:
...
-        tag: 1.4.4
+      # image:
+      #   tag: 1.4.4
     # See Global section to disable KSPM.
-    kspmAnalyzer:
-      image:
-        tag: 1.15.0
+    # kspmAnalyzer:
+    #   image:
+    #     tag: 1.15.0
```
Comment (shawnjohnson) [inline sysdig-values.yaml]:
```suggestion
```

File: sysdig-values.yaml [hunk 3]
```diff
@@ -50,28 +50,28 @@ nodeAnalyzer:
...
+      deploy: true
+      # image:
+      #   tag: 0.3.4
     # New engine image scanner
     runtimeScanner:
       deploy: true
-      image:
-        tag: 1.4.4
+      # image:
+      #   tag: 1.4.4
```
Comment (shawnjohnson) [inline sysdig-values.yaml]:
```suggestion
```

File: sysdig-values.yaml [hunk 4]
```diff
@@ -50,28 +50,28 @@ nodeAnalyzer:
...
+      # image:
+      #   tag: 0.1.22
     # New engine host scanner
     hostScanner:
-      deploy: false
-      image:
-        tag: 0.3.4
+      deploy: true
+      # image:
+      #   tag: 0.3.4
```
Comment (shawnjohnson) [inline sysdig-values.yaml]:
```suggestion
```

File: sysdig-values.yaml [hunk 5]
```diff
@@ -50,28 +50,28 @@ nodeAnalyzer:
...
+      # image:
+      #   tag: 0.1.14
     # Legacy engine image scanner
     imageAnalyzer:
-      deploy: true
-      image:
-        tag: 0.1.22
+      deploy: false
+      # image:
+      #   tag: 0.1.22
```
Comment (shawnjohnson) [inline sysdig-values.yaml]:
```suggestion
```

File: sysdig-values.yaml [hunk 6]
```diff
@@ -50,28 +50,28 @@ nodeAnalyzer:
       deploy: false
     # Legacy engine host scanner
     hostAnalyzer:
-      deploy: true
-      image:
-        tag: 0.1.14
+      deploy: false
+      # image:
+      #   tag: 0.1.14
```
Comment (shawnjohnson) [inline sysdig-values.yaml]:
```suggestion
```

**Review by shawnjohnson — CHANGES_REQUESTED** [review]

**Comment by alvarj84** [comment]
Thanks @shawnjohnson I added your suggestions.

**Pull request merged**